### DISCUSSION TITLE
Victron Pins sind gleich mit den Display Pins

### DISCUSSION THREAD

**Discussion by flofresser** [hoylabs/OpenDTU-OnBattery#92] [Q&A] [answered]
Hi helgeerbe!

Erstmal vorab, ich finde deinen OPENDTU fork genial.
Es funktioniert auch alles super.
Jetzt habe ich gesehen dass man bei OPENDTU auch ein Display anschließen kann.
Das Funktioniert auch super.

Aber soweit ich das verstehe werden für das Display und für die Victrons die gleichen Pins am ESP32 verwendet.
-DVICTRON_PIN_TX=21
-DVICTRON_PIN_RX=22

Ist bei deinem Fork das Display auch angedacht?
Oder würde das sogar schon funktionieren wenn ich die Pinzuweisung ändere?

lg

**Comment by helgeerbe** [comment, marked as answer]
Hi @flofresser, Display sollte funktionieren, wenn es in der originalen OpenDTU funktioniert. Natürlich musst du die entsprechenden PINs anpassen. Ich merge ca. jede Woche den aktuellen Stand. Du sieht auch unter Info->System->Firmware Update, ob dein System noch aktuell ist (gegenüber der Originalversion).
